ipatests/test_integration/test_ntp_replacement.py

```diff
@@ -0,0 +1,83 @@
+#
+# Copyright (C) 2018  FreeIPA Contributors see COPYING for license
+#
+
+from ipatests.test_integration.base import IntegrationTest
+from ipatests.pytest_ipa.integration import tasks
+
+
+class TestNTPreplacement(IntegrationTest):
+    """
+    Test NTP Replacement
+    """
+
+    def test_server_install_without_option_N(self):
+        """
+        test to verify ipa-server-install uses default chrony configuration
+        """
+        result = tasks.install_master(self.master, setup_dns=False)
+        assert result.returncode == 0
```

Comment:
``assert result.returncode == 0`` is only required if you use ``raiseonerr=False`` argument. Please remove it. It's just noise.

Reply:
removed